parktain/main.py

```diff
@@ -65,12 +65,23 @@ def checkins_reminder():
     date = datetime.now().strftime('%d %B, %Y')
     bot.speak('Morning! What are you doing on {}!'.format(date), "#checkins")
 
-@bot.listen_for(lambda user, channel, message: True)
+@bot.listen_for(lambda user, channel, message: True, target_channel='clickbaits')
```

Comment:
I'm adding an `ignore_channel` flag.  What if someone posts a link directly to `clickbaits`? Do we want to ignore that or do we also notify that?